cogniac/gateway.py

```diff
@@ -35,23 +50,88 @@ class CogniacGateway(object):
     media from another host on the same network.
     """
 
-    def __init__(self, timeout=60, url_prefix=None):
+    ##
+    #  get
+    ##
+    @classmethod
+    @retry(stop_max_attempt_number=8, wait_exponential_multiplier=500, retry_on_exception=server_error)
+    def get(cls,
+            connection=None,
+            gateway_id=None,
+            url_prefix=None):
+        """
+        get single network camera
+        connnection (CogniacConnection): Authenticated CogniacConnection object
+        gateway_id (String): the unique identifier of a Gateway object
+        returns CogniacGateway object
+        """
+        gateway_dict = None
+        if connection and gateway_id:
+            resp = connection._get("/gateways/{}".format(gateway_id))
+            gateway = resp.json()
```

Comment:
is there an exception that should be handled from the connection._get call?

Reply:
I thought this followed the existing pattern, but I'll verify.